NuRadioReco/modules/phasedarray/triggerSimulator.py

```diff
@@ -564,11 +564,12 @@ def run(self, evt, station, det,
                                       primary_angles=phasing_angles, trigger_delays=trigger_delays)
 
         trigger.set_triggered(is_triggered)
-
+        
         if is_triggered:
-            trigger.set_trigger_time(trigger_time)
+            #trigger_time(s)= time(s) from start of trace + start time of trace with respect to moment of first interaction = trigger time from moment of first interaction; time offset to interaction time (channel_trace_start_time) already recognized in self.phased_trigger
+            trigger.set_trigger_time(trigger_time)# 
             trigger.set_trigger_times(trigger_times)
-        else:
+	else:
```

Comment:
Hi @jannavischer, I hope you can see the test output? It seems you used spaces instead of tabs (or the other way round) here:
```
./NuRadioReco/modules/phasedarray/triggerSimulator.py:572:6: E999 TabError: inconsistent use of tabs and spaces in indentation
	else:
	    ^
1     E999 TabError: inconsistent use of tabs and spaces in indentation
1
Error: Process completed with exit code 1.```

which means this code doesn't run.